ui/src/app/services/fetch/fetch.service.ts

```diff
@@ -63,6 +63,13 @@ export class FetchService {
     return this.http.post<HttpResponse<IConv>>(`${this.url}/typemap/table?table=${tableName}`, data)
   }
 
+  restoreTable(tableName: string) {
+    return this.http.post<HttpResponse<IConv>>(`${this.url}/restore/table?table=${tableName}`, {})
+  }
+  dropTable(tableName: string) {
+    return this.http.put<HttpResponse<IConv>>(`${this.url}/drop/table?table=${tableName}`, {})
+  }
+
```

Comment:
We should use ids since we are anyway moving to ids to avoid the increase in work later

Reply:
tableName in parameter changed to tableId